cranelift/assembler/meta/src/dsl/encoding.rs

```diff
@@ -47,7 +47,7 @@ pub struct Rex {
     /// Any legacy prefixes that should be included with the instruction.
     pub prefixes: LegacyPrefixes,
     /// The opcode of the instruction.
-    pub opcode: u8,
+    pub opcode: u32,
```

Comment:
I do wonder if we should keep this as a `u8`; we don't want to have to write opcode sequences backwards like we do below in `or.rs`. Is it "kosher" with the manual if we create a new `LegacyPrefix::_660F`?

Reply:
Yeah, "escape opcode byte"... it's almost a `LegacyPrefix` but not quite. Maybe we should add an `.escape(0x0F)` to the DSL or something like that.